source/tests/chaos/readme.md

```diff
@@ -0,0 +1,16 @@
+# Chaos Tests
+
+This aims at testing the Navitia's dependency on [Chaos](https://github.com/CanalTP/chaos)
+
+## How to run
+Those tests are part of `docker_test` test suite, hence depend on the same requirements.
+If you haven't set any environment just yet, create a new virtual env with :
+
+```sh
+pip install -r ../../tyr/requirements.txt -r ../../eitri/requirements.txt
```

Comment:
You have to use tyr/requirements`_dev`.txt for pytest (and after a local test, eitri/requirements.txt isn't needed for chaos_tests).

So you can pick any of the 2 for the doc:
For a specific chaos_tests venv
```suggestion
pip install -r ../../tyr/requirements_dev.txt
```
Or for a whole docker_test venv
```suggestion
pip install -r ../../tyr/requirements_dev.txt -r ../../eitri/requirements.txt
```